File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnLayoutFormMap.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdLayoutForm;

import org.jetbrains.annotations.Nullable;

import java.util.Map;

@SuppressWarnings("unused")
public class DefnLayoutFormMap
{
  @Nullable
  public MetaIdLayoutForm asideDefaultLayoutId;

  public MetaIdLayoutForm[] keys;

  public Map<MetaIdLayoutForm, DefnLayoutForm> map;

  @Nullable
  public MetaIdLayoutForm mobileDefaultLayoutId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnLayoutFormMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdLayoutForm

interface DefnLayoutFormMap {
    val asideDefaultLayoutId: MetaIdLayoutForm?
    val keys: List<MetaIdLayoutForm>
    val map: Map<MetaIdLayoutForm, DefnLayoutForm>
    val mobileDefaultLayoutId: MetaIdLayoutForm?
}
